src/components/TheCanvas.vue

```diff
@@ -18,15 +18,15 @@
 <script lang="ts">
 import { Prop } from 'vue-property-decorator'
 import Component from 'vue-class-component'
-import { Tool, Tracker } from '@/tools/Tool'
+import { ToolInterface, Tracker } from '@/tools/Tool'
```

Comment:
I don't like naming an interface as interface. Is there a different name you could use?